posthog/clickhouse/migrations/0097_session_replay_events_v2_test.py

```diff
@@ -0,0 +1,17 @@
+from posthog.clickhouse.client.migration_tools import run_sql_with_exceptions
+from posthog.session_recordings.sql.session_replay_event_v2_test_sql import (
+    SESSION_REPLAY_EVENTS_V2_TEST_KAFKA_TABLE_SQL,
+    SESSION_REPLAY_EVENTS_V2_TEST_MV_SQL,
+    SESSION_REPLAY_EVENTS_V2_TEST_DATA_TABLE_SQL,
+    SESSION_REPLAY_EVENTS_V2_TEST_WRITABLE_TABLE_SQL,
+    SESSION_REPLAY_EVENTS_V2_TEST_DISTRIBUTED_TABLE_SQL,
+)
+
+
+operations = [
```

Comment:
Pawel, could you add the `SESSION_REPLAY_EVENTS_V2_TEST_DISTRIBUTED_TABLE_SQL` to the coordinator node as well? It's a recently introduced change that I'll show in the next all-hands demo, meanwhile, you can take a look [here](https://github.com/PostHog/posthog/blob/master/posthog/clickhouse/migrations/README.md).

If you can make the `SESSION_REPLAY_EVENTS_V2_TEST_DISTRIBUTED_TABLE_SQL` set the ON CLUSTER clause as optional, like it's done [here](https://github.com/PostHog/posthog/blob/master/posthog/session_recordings/sql/session_replay_event_sql.py#L210)

The change would only be adding this (the rest of the lines you added would be left as you listed them):

```suggestion
run_sql_with_exceptions(SESSION_REPLAY_EVENTS_V2_TEST_DISTRIBUTED_TABLE_SQL(on_cluster=False), node_role=NodeRole.COORDINATOR),
```

Reply:
Yes, of course, thanks for pointing this out.